rales_engine/app/controllers/api/v1/items_controller.rb

```diff
@@ -0,0 +1,32 @@
+module Api
+  module V1
+    class ItemsController < ApplicationController
+
+      def index
+        render json: Item.all
+      end
+
+      def show
+        render json: Item.find(params[:id])
+      end
+
+      def create
```

Comment:
@markyv18 @brandonrandall we don't need this method along with `update` and `destroy` because the specs don't request this functionality!

same with the `InvoicesController`